src/views/Package/PackageLayout.tsx

```diff
@@ -31,33 +31,30 @@ export const PackageLayout: FunctionComponent = () => {
     pageTitle,
   } = usePackageState();
 
+  const [tabIndex, setTabIndex] = useState(0);
+
+  useEffect(() => {
+    setTabIndex(0);
+  }, [pageTitle]);
```

Comment:
This is to reset the selected tab to docs when a user clicks on a dependency or link to another package